OpenRA.Mods.Common/Scripting/Properties/GeneralProperties.cs

```diff
@@ -89,6 +89,12 @@ public GeneralProperties(ScriptContext context, Actor self)
 		[Desc("The actor position in cell coordinates.")]
 		public CPos Location { get { return Self.Location; } }
 
+		[Desc("The x value of the actor position.")]
+		public int X { get { return Self.Location.X; } }
```

Comment:
My bad, I was only using the Lua API as reference and didn't think about trying it.